### PR TITLE
Backported to API 23

### LINKED ISSUE #34: Consider minSdk lower than 24.
I don't feel like doing this... but someone might.

### PR DESCRIPTION
**Pull request by fercarcedo**
Closes #34

### CODE REVIEW

File: build.gradle [hunk 1]
```diff
@@ -53,6 +53,7 @@ buildscript {
               'design': "com.android.support:design:${versions.supportLibrary}",
               'constraintLayout': 'com.android.support.constraint:constraint-layout:1.1.0-beta4',
           ],
+          'threetenabp': 'com.jakewharton.threetenabp:threetenabp:1.0.5',
```
Comment (JakeWharton) [inline build.gradle]: This isn't an Android library so it shouldn't be under 'android', but I'd prefer not to include it altogether: https://github.com/JakeWharton/SdkSearch/pull/38

File: frontend/android/src/main/res/layout/item.xml [hunk 2]
```diff
@@ -6,27 +6,27 @@
...
-    <TextView
+    <android.support.v7.widget.AppCompatTextView
         android:id="@+id/package_name"
         android:layout_width="0dp"
         android:layout_height="wrap_content"
         android:layout_marginEnd="8dp"
         android:layout_marginStart="16dp"
         android:layout_marginTop="12dp"
-        android:autoSizeTextType="uniform"
+        app:autoSizeTextType="uniform"
```
Comment (JakeWharton) [inline frontend/android/src/main/res/layout/item.xml]: This should be sorted. This project uses https://github.com/square/java-code-styles which has the correct formatting rules for XML.

**Comment by fercarcedo** [comment]
I have formatted the XML with the code style and removed the threetenbp library. The branch is also rebased now.

**Comment by JakeWharton** [comment]
Thanks!

**Pull request merged**